api/bases/glance.openstack.org_glanceapis.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.10.0
+    controller-gen.kubebuilder.io/version: v0.9.2
```

Comment:
you remove your local bin/controller-gen to get the new version installed on next run

Reply:
> you remove your local bin/controller-gen to get the new version installed on next run

Phew, I had proposed a change to cleanup old binaries elsewhere[1] , but you know I've to manually ensure that:/ 

[1] https://github.com/openstack-k8s-operators/keystone-operator/pull/179